luigi/file.py

```diff
@@ -35,6 +35,9 @@ class atomic_file(AtomicLocalFile):
     Also cleans up the temp file if close is not invoked
     """
 
+    def __init__(self, path, mode):
+        super(AtomicLocalFile, self).__init__(path, mode)
+
```

Comment:
What happens if you remove this? Can't you still initialize this atomic_file class, since it's just AtomicLocalFile with an additional method (move_to_final_destination)?